### ISSUE TITLE
TBS Smartaudio power management always sets level 2

### ISSUE THREAD

**Issue by Wikiwix** [flyduino/kissfcv2-firmware#8]
With various hardware combinations using Smartaudio does work for setting Channel and Band, but regardless of the Output power setting sets to power level 2 (used the standard 25mW 25mW as well as 0mW 0mW or 1mW 1mW). I checked the setting via the LEDs on the VTX and during a race with a power meter (after I got singled out for having too high power…)
During above mentioned race someone else told me that this is not working in recent KISS versions for him as well.
As for combinations:
KISS v2 and TBS Unify Race (locked and unlocked)
FETtec FC and TBS Unify Race (locked and unlocked)
FETtec FC and TBS UNIFY Nano Pro32 (locked and unlocked)

**Comment by lichtl** [comment]
Hi,
on which firmware you have been exactly?

**Comment by Wikiwix** [comment]
Sorry for omitting that crucial piece of information.
I am on 1.3RC40o

**Comment by lichtl** [comment]
Will have a look today.

**Comment by Wikiwix** [comment]
New additional info:
I have one FETtec FC 1.0 (the other ones are revision 0.3) together with a Unify Nano Pro32 which does not face the issue. Just checked again changing to 100mW set the VTX to level 2 and setting back to 25 put the VTX back to level 1.

**Comment by Wikiwix** [comment]
Are there any news on this?
I guess the next race is still a bit off 🙈, but would be great to have that fixed by then.

**Comment by Wikiwix** [comment]
@lichtl this is still an issue. A friend ran into it with a FETtec FC and an original Unify Nano.
Could you please investigate this?
If there is anything I can help with feel free to ask

**Comment by lichtl** [comment]
Hi. Sorry this should be fixed with next release which will be released shortly. Can you give it a try. Here is the FC/GUI https://www.dropbox.com/sh/v0jnrj0prabkxxb/AADDZbBx24QtCIi-4zKI8jVya?dl=0

**Comment by Wikiwix** [comment]
@lichtl Thanks. I will try it today. Anything else I need to know for the update? How stable is it? Any issues to be aware of?

**Comment by lichtl** [comment]
No known issues from multiple testers. This is the planned release candidate. However best is to not import old config and just apply changes manually.

**Comment by Wikiwix** [comment]
@lichtl one has to use the level and not the mWatt for it to work (25mW still leads to poer level 2 on a Unify Race) but setting this value to 1 sets the power level to one 👍 So I would suggest to change the description in the KISS GUI and the defaults of the values (to 1 instead of 25) to match that behaviour.
Changes to the PIDs i.e. do I have to tune again? 🙈 
And anything new besides the RTH function?

**Comment by lichtl** [comment]
The situation there is that various smartaudio implementations handle this different (even between smartaudio, 2.0/2,1 and the new nano32). Maybe worth put a note next to it.
PIDs should be unchanged there was some bugfixes but nothing that should have a big impact on existing tune.

**Comment by Wikiwix** [comment]
@lichtl just to be sure: A config from an already updated FC can be used without concerns for the new Firmware?

**Comment by lichtl** [comment]
yes. Should be no problem. Just a note there was one more bug in this but only affect BF ESC orientation. (resolved in link)
